subt/docker/robotika/run_solution.bash

```diff
@@ -38,16 +38,19 @@ else
     LAUNCH_FILE="proxy sim.launch"
     CONFIG_FILE="zmq-subt-x2.json"
 fi
+
 echo "Starting ros<->osgar proxy"
 # Wait for ROS master, then start ros nodes
 # http://wiki.ros.org/roslaunch/Commandline%20Tools#line-45 
 roslaunch $LAUNCH_FILE --wait robot_name:=$ROBOT_NAME &
 
 /osgar-ws/src/osgar/subt/cloudsim2osgar.py $ROBOT_NAME &
 
+CONFIG_FILE=/osgar-ws/src/osgar/subt/$CONFIG_FILE
+LOG_FILE=/osgar-ws/logs/zmq-subt-x2-$(date +%Y-%m-%dT%H.%M.%S).log
```

Comment:
well, the log files should reflect used config file?? (here you have two options x2 and x4)

Reply:
good catch :+1: thanks